### ISSUE TITLE
What is a best way to contribute as translator?

### ISSUE THREAD

**Issue by tkoyama010** [openjournals/joss#906]
I would like to translate this excellent journal articles into my native language. Is there a way to contribute as translator in this community? I could not find it in [document](https://joss.readthedocs.io/en/latest/index.html) .

**Comment by danielskatz** [comment]
I don't think we have an answer to this, or at least, we haven't thought about it very much.
